unified/install.sh

```diff
@@ -88,6 +95,22 @@ if [ -z "$prefix" ]; then
     fi
 fi
 
+if [ -z "$python3" ]; then
+    if $nonroot; then
+        python3=~/scylladb/python3/bin/python3
+    else
+        python3=/opt/scylladb/python3/bin/python3
+    fi
+fi
+
+if [ -z "$sysconfdir" ]; then
+    if [ "$ID" = "ubuntu" ] || [ "$ID" = "debian" ]; then
+        sysconfdir=/etc/default
+    else
+        sysconfdir=/etc/sysconfig
+    fi
+fi
+
```

Comment:
How does this work in nonroot mode?

Reply:
In nonroot mode, $rprefix (real patch of prefix) will be added for access sysconfig files.
So it's not a problem.

Reply:
For example, in install.sh (https://github.com/scylladb/scylla/blob/master/install.sh#L186), it get the final real paths by following code:

```
if ! $nonroot; then
    retc="$root/etc"
    rsysconfdir="$root/$sysconfdir"
    rsystemd="$rusr/lib/systemd/system"
    ....
else
    retc="$rprefix/etc"
    rsysconfdir="$rprefix/$sysconfdir"
    rsystemd="$retc/systemd"
    ....
fi
```

Reply:
Yes.

Reply:
We should move away from /etc/sysconfig and /etc/default and put all such files into /etc/scylla. But not in this series.